Build error: innerFSMock.readFileSync is possibly 'undefined'

### Summary

Following the steps in `CONTRIBUTING.md`, `npm install` and `npm test`:

```
> stryker-webpack-transpiler@0.1.4 build C:\Code\stryker\packages\stryker-webpack-transpiler
> tsc

test/unit/fs/InputFileSystemSpec.ts(119,7): error TS2532: Object is possibly 'undefined'.
test/unit/fs/InputFileSystemSpec.ts(137,7): error TS2532: Object is possibly 'undefined'.
```

--- a/packages/stryker-webpack-transpiler/test/unit/fs/InputFileSystemSpec.ts
+++ b/packages/stryker-webpack-transpiler/test/unit/fs/InputFileSystemSpec.ts
@@ -2,12 +2,12 @@ import { expect } from 'chai';
 import MemoryFS, * as memoryFSModule from '../../../src/fs/MemoryFS';
 import InputFileSystem from '../../../src/fs/InputFileSystem';
 import { Mock, createMockInstance } from '../../helpers/producers';
-import { AbstractInputFileSystem, CachedInputFileSystem } from 'enhanced-resolve';
+import { CachedInputFileSystem } from 'enhanced-resolve';
 
 describe('InputFileSystem', () => {
   let sut: InputFileSystem;
   let memoryFSMock: Mock<MemoryFS>;
-  let innerFSMock: Mock<AbstractInputFileSystem>;
+  let innerFSMock: Mock<CachedInputFileSystem>;
 
   beforeEach(() => {
     memoryFSMock = createMockInstance(MemoryFS);

